va/validation-authority_test.go

```diff
@@ -541,6 +541,36 @@ func TestValidateHTTP(t *testing.T) {
 	test.AssertEquals(t, core.StatusValid, mockRA.lastAuthz.Challenges[0].Status)
 }
 
+func TestValidateHTTPResponseDocument(t *testing.T) {
+	chall := core.HTTPChallenge01(accountKey)
```

Comment:
Now that you've extracted the logic into truncateBody, this test is more complex than it needs to be. You can turn it into TestTruncateBody and just call the function directly for various inputs (nil, short, long, long with multibyte character spanning the truncate point).